src/storages/pluggable/SplitsCachePluggable.ts

```diff
@@ -0,0 +1,274 @@
+   * or rejected with an SplitError if it fails (e.g., wrapper operation fails)
+   */
+  addSplits(entries: [string, string][]): Promise<boolean[]> {
+    return Promise.all(entries.map(keyValuePair => this.addSplit(keyValuePair[0], keyValuePair[1])));
+  }
+
+  /**
+   * Remove a given split.
+   * The returned promise is resolved when the operation success, with a boolean indicating if the split existed or not.
+   * or rejected with an SplitError if it fails (e.g., wrapper operation fails).
+   */
+  removeSplit(name: string): Promise<boolean> {
+    return this.getSplit(name).then((split) => {
+      if (split) {
+        const parsedSplit = JSON.parse(split);
+        this._decrementCounts(parsedSplit);
+      }
+      return this.wrapper.del(this.keys.buildSplitKey(name));
+    });
+  }
+
+  /**
+   * Remove a list of splits.
+   * The returned promise is resolved when the operation success, with a boolean array indicating if the splits existed or not.
+   * or rejected with an SplitError if it fails (e.g., wrapper operation fails).
+   */
+  removeSplits(names: string[]): Promise<boolean[]> {
+    return Promise.all(names.map(name => this.removeSplit(name)));
+  }
+
+  /**
+   * Get split.
+   * The returned promise is resolved with the split definition or null if it's not defined,
+   * or rejected with an SplitError if wrapper operation fails.
+   */
+  getSplit(name: string): Promise<string | null> {
+    return this.wrapper.get(this.keys.buildSplitKey(name));
+  }
+
+  /**
+   * Get list of splits.
+   * The returned promise is resolved with a map of split names to their split definition or null if it's not defined,
+   * or rejected with an SplitError if wrapper operation fails.
+   */
+  getSplits(names: string[]): Promise<Record<string, string | null>> {
+    const keys = names.map(name => this.keys.buildSplitKey(name));
+
+    return this.wrapper.getMany(keys).then(splitDefinitions => {
+      const splits: Record<string, string | null> = {};
+      names.forEach((name, idx) => {
+        splits[name] = splitDefinitions[idx];
+      });
+      return Promise.resolve(splits);
+    });
+  }
+
+  /**
+   * Get list of all split definitions.
+   * The returned promise is resolved with the list of split definitions,
+   * or rejected with an SplitError if wrapper operation fails.
+   */
+  getAll(): Promise<string[]> {
+    return this.wrapper.getKeysByPrefix(this.keys.buildSplitKeyPrefix()).then(
+      (listOfKeys) => Promise.all(listOfKeys.map(this.wrapper.get) as Promise<string>[])
+    );
+  }
+
+  /**
+   * Get list of split names.
+   * The returned promise is resolved with the list of split names,
+   * or rejected with an SplitError if wrapper operation fails.
+   */
+  getSplitNames(): Promise<string[]> {
+    return this.wrapper.getKeysByPrefix(this.keys.buildSplitKeyPrefix()).then(
+      (listOfKeys) => listOfKeys.map(this.keys.extractKey)
+    );
+  }
+
+  /**
+   * Check traffic type existance.
```

Comment:
Typo: `existence`

Reply:
thanks Emi. We had that typo in other places too. Will be fixed